Doc/library/sqlite3.rst

```diff
@@ -1223,6 +1250,42 @@ Connection objects
 
       .. versionadded:: 3.11
 
+   .. attribute:: autocommit
+
+      This attribute controls :pep:`249`-compliant transaction behaviour.
+      *autocommit* has three allowed values:
+
+      * ``False``: select :pep:`249`-compliant transaction behaviour,
+        implying that :mod:`!sqlite3` ensures a transaction is always open.
+        Use :meth:`~Connection.commit` and :meth:`~Connection.rollback` to
+        close transactions.
+
+        This is the recommended value of *autocommit*.
+
+      * ``True``: Use SQLite's `autocommit mode`_.
+        :meth:`~Connection.commit` and :meth:`~Connection.rollback`
+        have no effect in this mode.
+
+      * :data:`COMPAT_TRANSACTIONAL_CONTROL`:
+        Pre-Python 3.12 (non-:pep:`249`-compliant) transaction control.
+        See :attr:`isolation_level`.
+
+        This is currently the default value of *autocommit*.
+
+      Changing *autocommit* to ``False`` will open a new transaction,
+      and changing it to ``True`` will commit any pending transaction.
+
+      See :ref:`sqlite3-transaction-control-autocommit` for more details.
+
+      .. note::
+
+         The :pep:`249`-compliant autocommit feature and the
+         SQLite `autocommit mode`_ are two related, but different concepts.
```

Comment:
```suggestion
         SQLite `autocommit mode`_ are two related but distinct concepts.
```

Like some others, I made this suggestion before to improve readability and sound more idiomatic, but it seems it wasn't applied completely...another pitfall of manually trying to apply suggestions, which is why I do a full re-review in that case to catch these omissions.